source/_reference/methods.coffee

```diff
@@ -511,25 +539,50 @@ module.exports = exports =
       httpOnly: true
       http: "GET /events/{id}/{fileId}[/{fileName}]"
       description: """
-                   Gets the attached file. Accepts an arbitrary filename path suffix (ignored) for easier link creation.
+                   Gets the attached file. Accepts an arbitrary filename path suffix (ignored) for easier link readability.
+                   For this function using the `auth` query parameter is not accepted. You can either use the [access token](##{dataStructure.getDocId("access")}) in the `Authorization` header or provide the `readToken` query parameter.
                    """
       params:
         properties: [
+          key: "id"
+          type: "[identifier](##{dataStructure.getDocId("identifier")})"
+          http:
+            text: "set in request path"
+          description: """
+                       The id of the event.
+                       """
+        ,
+          key: "fileId"
+          type: "[identifier](##{dataStructure.getDocId("identifier")})"
+          http:
+            text: "set in request path"
+          description: """
+                       The id of the attached file.
+                       """
+        ,
+          key: "fileName"
+          type: "string"
+          optional: true
+          http:
+            text: "set in request path"
+          description: """
+                       The name of the file.
+                       """
+        ,
```

Comment:
Similarly I'd just remove that because it's not consistent with how the rest of the doc describes method parameters, and therefore confusing. To propose something more appropriate I need to know: how has line 540 above been proven insufficient to document the method? (It's using the same notation as the rest of the doc so I'm surprised)

Reply:
I don't see the inconsistency. Methods update and delete methods mention the path parameters such as `id` in the params section of the method.

Reply:
Sorry I wasn't precise enough. Those other methods you mention have to include path params such as `id` in the params list because that's how they are called with Socket.IO (for which the URL path isn't used). For HTTP-only methods such as this one (admittedly there are only three) we didn't list path params in the params, I don't see why we should and I think I see why we shouldn't. As said above, if you want me to help more please first tell me what the issue is currently.

Reply:
Thank you for clearing out that the parameters section adds some description to the path parameters when socket.IO support exists, but I didn't find this information. Should we state this in the [calling with websockets](http://api.pryv.com/reference/#calling-with-websockets) section?

Reply:
Till now I thought things were pretty self-explanatory: if you use the doc for HTTP calls you'll just check the HTTP method and path and see that params part of the path are listed with the mention 'HTTP: set in request path'; if you use it for socket calls you'll just ignore the HTTP labels and everything is fine. But from this conversation I see I might be wrong; what may help decide is to know whether this was an issue for anyone else using our doc?

Reply:
I figured it out, but was bothered by this as I considered this as an inconsistency.